hack/deploy/configure-values.sh

```diff
@@ -35,13 +35,18 @@ if [[ ! -v VNET_SUBNET_ID ]]; then
     VNET_SUBNET_ID=$(jq -r ".subnets[0].id" <<< "$VNET_JSON")
 fi
 
+# The // empty ensures that if the files is 'null' or not prsent jq will output nothing
+# If the value returned is none, its from jq and not the aks api in this case we return ""
+NETWORK_PLUGIN=$(jq -r ".networkProfile.networkPlugin // empty | if . == \"none\" then \"\" else . end" <<< "$AKS_JSON")
+NETWORK_PLUGIN_MODE=$(jq -r ".networkProfile.networkPluginMode // empty | if . == \"none\" then \"\" else . end" <<< "$AKS_JSON")
+NETWORK_POLICY=$(jq -r ".networkProfile.networkPolicy // empty | if . == \"none\" then \"\" else . end" <<< "$AKS_JSON")
```

Comment:
There is probably a better way to do this thats less confusing. Didnt find it however so I have this but Im not in love with it. If anyone has more experience here please chime in.